test/integration/load_stats_integration_test.cc

```diff
@@ -379,6 +434,7 @@ class LoadStatsIntegrationTest : public Grpc::GrpcClientIntegrationParamTest,
   uint32_t load_requests_{};
   EdsHelper eds_helper_;
   bool locality_weighted_lb_{};
+  bool drop_overload_test_{false};
```

Comment:
If possible, try to remove this global state. It makes it a bit more concise when the entire change is part of the test itself rather than some global state that one needs to look carefully through to understand the impact.

Reply:
done